### ISSUE TITLE
RAW: Add RawTherapee flag to use existing sidecar files

### ISSUE THREAD

**Issue by rizmot** [photoprism/photoprism#1267]
Currently, RawTherapee ignores pre-existing sidecar-profiles. This can be changed using the `-s` command-line parameter according to https://rawpedia.rawtherapee.com/Command-Line_Options#RawTherapee_CLI

> True, I didn't add / change any RawTherapee parameters. From what I remember, the standard profile is used by default. Probably need to modify the command to use other profiles / sidecars. Is there a generic way to do this or do we need to add yet another config parameter to set a profile name / type?

So i tested the following command: `rawtherapee-cli -d -p raw.pp3 -s -o output.jpg -c input.RAW`
This will use the default ( `-d` ) profile, then if another profile is specified ( `-p raw.pp3` ) it will be stacked on top of the default profile. Then, if the `-s` command is set, a sidecar-profile, if one is found, will be stacked on top of whatever was specified before. The `-d -p -s` parameters have to be set in the right order since the last one will overwrite the ones before. As I understand it, `-d` and `-p raw.pp3` might be optional in this case. So `rawtherapee-cli -s -o output.jpg -c input.RAW` should do the trick.

_Originally posted by @rizmot in https://github.com/photoprism/photoprism/issues/1245#issuecomment-830635711_

**Comment by lastzero** [comment]
@rizmot Started a new preview build with this:

https://docs.photoprism.app/getting-started/updates/#development-preview

Can you test it please? If it works, we're going to release next week.

**Comment by lastzero** [comment]
Updated Release Notes: https://docs.photoprism.org/release-notes/

**Comment by lastzero** [comment]
Already released it, hopefully it works for you as expected. At least it didn't seem to break anything. CR3 files still work.

**Comment by rizmot** [comment]
> @rizmot Started a new preview build with this: https://docs.photoprism.app/getting-started/updates/#development-preview
> 
> Can you test it please? If it works, we're going to release next week.

I tested it and it works fine. Pre-existing profiles are now used for RAW-Processing with RawTherapee. Thank you.

**Comment by lastzero** [comment]
Excellent, thanks for testing! :pray:

**Comment by louwers** [comment]
I can't seem to get this to work. The JPG output does not match that of Rawtherapee. The `.pp3` sidecar file is present and visible as one of the files:

![image](https://user-images.githubusercontent.com/649392/145695778-f2b4d53a-b95b-4f71-b2d0-460f79f76ed9.png)

I've disabled Darktable to ensure RAW processing is done by Rawtherapee:

![image](https://user-images.githubusercontent.com/649392/145695771-9f5e6875-2a72-4de1-b382-683390272608.png)

I've also tried re-indexing to no avail. 

Is there anything else I need to do to set this up?
